core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/config/LangChain4jConfig.java

```diff
@@ -31,6 +32,11 @@ public interface LangChain4jConfig {
     @ConfigDocDefault("10s")
     Optional<Duration> timeout();
 
+    /**
+     * Global temperature for LLM APIs
```

Comment:
Should we say what the allowed range is?

(And here we are, as I expressed my concern about creating similar abstractions - are we sure that all models and model providers will have the same allowed range for temperature?)

Reply:
Then it's questionable whether to have this abstraction at all, but ok, a different number range isn't that hardcore, I could imagine worse :)